nexus/src/lib.rs

```diff
@@ -41,10 +40,6 @@ use std::net::{SocketAddr, SocketAddrV6};
 use std::sync::Arc;
 use uuid::Uuid;
 
-// These modules used to be within nexus, but have been moved to
-// nexus-db-queries. Keeping these around temporarily for migration reasons.
-pub use nexus_db_queries::{authn, authz, db};
```

Comment:
Ooh, thanks for addressing this. rust-analyzer helps a lot, but I always find the "the thing I'm importing is actually a re-export of something else" a little confusing when I run into them.

Reply:
Totally! I think the intermediate transition also made sense, but as these are becoming more established as separate crates, it makes more and more sense to keep detangling 'em.